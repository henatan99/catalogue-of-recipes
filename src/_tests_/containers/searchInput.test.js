import React from 'react';
import { Provider } from 'react-redux';
import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import SearchInput from '../../containers/searchInput';
import createTestStore from './testStore';
describe('SearchInput', () => {
  let store;
  beforeEach(() => {
    store = createTestStore();
  });
  test('should match with snapshot', () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <MemoryRouter>
            <SearchInput />
          </MemoryRouter>
        </Provider>,
      ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});