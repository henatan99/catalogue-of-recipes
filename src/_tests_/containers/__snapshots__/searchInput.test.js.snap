// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AdminItemList should match with snapshot 1`] = `
<div
  className="form-wrapper"
>
  <form
    className="search-form"
    onSubmit={[Function]}
  >
    <input
      className="search-input"
      onChange={[Function]}
      type="text"
      value=""
    />
  </form>
</div>
`;

exports[`SearchInput should match with snapshot 1`] = `
<div
  className="form-wrapper"
>
  <form
    className="search-form"
    onSubmit={[Function]}
  >
    <input
      className="search-input"
      onChange={[Function]}
      type="text"
      value=""
    />
  </form>
</div>
`;
